test/pkgcheck.sh

```diff
@@ -81,11 +81,6 @@ Darwin)
   sysctl -n machdep.cpu.features
   sysctl -n machdep.cpu.leaf7_features
   sysctl -n machdep.cpu.extfeatures
-  CMAKE_ARGS="-DCMAKE_INSTALL_LIBDIR=lib -DPKGCONFIG_INSTALL_DIR=/lib/pkgconfig -DWITH_RPATH=on ${CMAKE_ARGS}"
-  CONFIGURE_ARGS="--libdir=lib ${CONFIGURE_ARGS}"
```

Comment:
Not sure how this ever passed tests? `--libdir=lib` with `make DESTDIR=hello/` puts the libraries in `hello/lib/*` instead of `hello/${prefix}/lib/*`.

Reply:
This worked because `Makefile` doesn't add directory separator after `prefix`. This means that `prefix` also had to always end with directory separator.